### PR TITLE
WSClient cleanups

### PR DESCRIPTION
**Pull request by mairas**
Two minor cleanups: first, token validity is now tested on a URL that doesn't return a ton of data. Second, a bit of logic cleanup that should have no functional changes.

### CODE REVIEW

File: src/net/ws_client.cpp [hunk 1]
```diff
@@ -320,7 +320,7 @@ void WSClient::test_token(const String server_address,
   HTTPClient http;
 
   String url = String("http://") + server_address + ":" + server_port +
-               "/signalk/v1/api/";
+               "/signalk/";
```
Comment (ba58smith) [inline src/net/ws_client.cpp:323]: THANK YOU!

**Review by ba58smith — APPROVED** [review]
This one's over my head. Sorry.

**Comment by mairas** [comment]
> This one's over my head. Sorry.

It absolutely isn't! :-D The diff just looks messy but all I did was change:
```
if (foo) {
  a_lot_of_code_here;
} else {
 a_bit_of_code_here;
}
```

to 

```
if (!foo) {
  a_bit_of_code_here;
  return;
}
a_lot_of_code_here;
```

File: src/net/ws_client.cpp [hunk 2]
```diff
@@ -257,60 +257,60 @@ void WSClient::connect() {
...
-    if (this->server_address.isEmpty()) {
-      if (!get_mdns_service(server_address, server_port)) {
-        debugE("No Signal K server found in network when using mDNS service!");
-      } else {
-        debugI("Signal K server has been found at address %s:%d by mDNS.",
-               server_address.c_str(), server_port);
-      }
-    }
+  String server_address = this->server_address;
+  uint16_t server_port = this->server_port;
```
Comment (ba58smith) [inline src/net/ws_client.cpp:272]: This is an exellent example of where adding the trailing underscore to member variables can simplify things:
```
String server_address = server_address_;
  uint16_t server_port = server_port_;
  ```
  Voila! No more confusion about which server_address or sever_port we're talking about!
  
  Just an observation. :-)

Comment (mairas) [reply src/net/ws_client.cpp:272]: Yeah, I've actually grown to like the underscores. I can add a cleanup commit to the PR to rename the remaining offending variables in WSClient...

**Pull request merged**